static/app/views/ddm/summaryTable.tsx

```diff
@@ -0,0 +1,112 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+import {formatMetricsUsingUnitAndOp, getNameFromMRI} from 'sentry/utils/metrics';
+import {Series} from 'sentry/views/ddm/metricsExplorer';
+
+export function SummaryTable({
+  series,
+  operation,
+  onClick,
+}: {
+  onClick: (seriesName: string) => void;
+  series: Series[];
+  operation?: string;
+}) {
+  return (
+    <SummaryTableWrapper>
+      <HeaderCell />
+      <HeaderCell>Name</HeaderCell>
+      <HeaderCell>Avg</HeaderCell>
+      <HeaderCell>Min</HeaderCell>
+      <HeaderCell>Max</HeaderCell>
+      <HeaderCell>Sum</HeaderCell>
```

Comment:
Let's translate these and start using the translate function from now on on everything. I fixed all occurrences that were without so we should have a clean slate.